src/pytest_inmanta_lsm/plugin.py

```diff
@@ -69,15 +70,30 @@ def pytest_addoption(parser):
         dest="inm_lsm_noclean",
         help="Don't cleanup the orchestrator after tests (for debugging purposes)",
     )
+    group.addoption(
```

Comment:
Also mention the environment variable that can be used to set this setting to the help message.